adt-pulse-mqtt/package.json

```diff
@@ -4,8 +4,12 @@
   "description": "Home Assistant ADT Pulse Bridge using MQTT",
   "author": "ADT Pulse MQTT Team",
   "main": "server.js",
+  "engines": {
+    "node": ">= 18"
+  },
```

Comment:
Codifying a requirement already existing in the code, but one that isn't obvious until runtime if using an older engine.  This way, if your `node`  version is older, it will fail earlier and let you know.